test/LeanCode.Test.Helpers/LongRunningFact.cs

```diff
@@ -1,19 +1,16 @@
-using System;
 using Xunit;
```

Comment:
Is that namespace even still supported by the v3 version? Don't you have some older xunit creeped in somewhere, that is compiles?

Reply:
Yes, the [`FactAttribute`](https://github.com/xunit/xunit/blob/main/src/xunit.v3.core/FactAttribute.cs#L6) is still in `Xunit` namespace. :)